### DISCUSSION TITLE
maybe Bug

### DISCUSSION THREAD

**Discussion by MrTabaOfficial** [lavalink-devs/Lavalink#870] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/121251795/234427836-fea4821e-be26-405f-a378-ad6876226d96.png)

when i'm turning music It Starts playing and In 0 seccond Turning off its doing again and again

**Comment by topi314** [comment]
you don't provide enough information for us to say anything
if you update your post with the full logs then we might be able to help

**Comment by Tomato6966** [comment, marked as answer]
Lavalink does not execute things via the websocket.

These are execution logs, from your client. means you told lavalink to execute "stop" (aka stop the track, if you have a queue system on the client, this is ment to skip in the queue and request the next song)
and then you executed "destroy" which tells lavalink to delete the player.

I assume, that you have on your client somehow said to skip the track, and then destroyed the player.
